pkg/cmd/node/node_ssh.go

```diff
@@ -0,0 +1,142 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/dcos/dcos-cli/pkg/dcos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/dcos/dcos-core-cli/pkg/sshclient"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+func newCmdNodeSSH(ctx api.Context) *cobra.Command {
+	var leader, masterProxy bool
+	var mesosID, proxyIP, privateIP, user, config string
+	var options []string
+
+	cmd := &cobra.Command{
+		Use:   "ssh <command>",
+		Short: "Establish an SSH connection to the master or agent nodes of your DC/OS cluster",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			host, err := detectHost(leader, mesosID, privateIP)
+			if err != nil {
+				return err
+			}
+
+			proxy, err := detectProxy(ctx, masterProxy, proxyIP)
+			if err != nil {
+				return err
+			}
+
+			sshClient, err := sshclient.NewClient(
+				sshclient.ClientOpts{
+					Input:      ctx.Input(),
+					Out:        ctx.Out(),
+					ErrOut:     ctx.ErrOut(),
+					SSHOptions: options,
```

Comment:
maybe declare the ClientOpts variable before the command? then we don't have to add extra variables for flag parsing : 

    cmd.Flags().StringArrayVar(&opts.SSHOptions, "option", nil, "The SSH options")